Open Judge System/Workers/OJS.Workers.ExecutionStrategies/CSharpProjectTestsExecutionStrategy.cs

```diff
@@ -172,6 +200,9 @@ public override ExecutionResult Execute(ExecutionContext executionContext)
 
         private void CorrectProjectReferences(IEnumerable<TestContext> tests, Project project)
         {
+            File.WriteAllText(this.SetupFixturePath, SetupFixtureTemplate);
+            project.AddItem("Compile", $"{SetupFixtureFileName}.cs");
+
             foreach (var testName in this.TestNames)
             {
                 project.AddItem("Compile", $"{testName}.cs");
```

Comment:
Keep consistency with the project, add a runtime constant in the GlobalConstants for the file extension and reuse it.